src/win/util.c

```diff
@@ -1804,7 +1804,9 @@ int uv_os_uname(uv_utsname_t* buffer) {
     pRtlGetVersion(&os_info);
   } else {
     /* Silence GetVersionEx() deprecation warning. */
+    #if _MSC_VER
```

Comment:
Elsewhere we use `#ifdef`, I think:

```suggestion
    #ifdef _MSC_VER
```